refiner/tests/test_service_refine.py

```diff
@@ -161,3 +170,60 @@ def test_results_section_minimal_on_nonexistent_code(sample_xml_files):
     )[0]
     assert results_section.get("nullFlavor") == "NI"
     assert not results_section.xpath(".//hl7:entry", namespaces=NAMESPACES)
+
+
+def test_refine_rr_by_condition(sample_xml_files):
```

Comment:
@robertmitchellv the two tests I could think of was to check for a 1) RRVS1-tagged observation and 2) that a non-matching JID filters out everything, in addition to the schematron test up above. Not thrilled with how robust these tests are give our test data set, but lmk if you can think of something to improve